FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_light_weight/DemoTasks/LightWeightMQTTExample.c

```diff
@@ -944,47 +1091,65 @@ static void prvMQTTProcessIncomingPacket( Socket_t xMQTTSocket )
 
     /* Determine incoming packet type and remaining length. */
     xNetworkContext.xTcpSocket = xMQTTSocket;
+
     xResult = MQTT_GetIncomingPacketTypeAndLength( prvTransportRecv,
```

Comment:
Does this need to be retried as you have added a timeout for socket?

Reply:
hmm that is a good point so I will just remove the timeout in that case